admin-manual/installation-setup/customization/antivirus-admin.rst

```diff
@@ -61,43 +61,43 @@ be recorded.
 
 From the `ClamAV manual pages`_::
 
-	MaxFileSize SIZE
-		Files  larger than this limit won't be scanned. Affects the input file
-		itself as well as files contained inside it (when the input file is an
-		archive, a document or some other kind of container).
+    MaxFileSize SIZE
+        Files  larger than this limit won't be scanned. Affects the input file
+        itself as well as files contained inside it (when the input file is an
+        archive, a document or some other kind of container).
 
-	MaxScanSize SIZE
-		Sets the maximum amount of data to be scanned for each input file.
-		Archives and other containers are recursively extracted  and  scanned
-		up to this value. The size of an archive plus the sum of the sizes of
-		all files within archive count toward the scan size. For example, a
-		1M uncompressed archive containing a single 1M inner file counts as
-		2M toward the max scan size.
+    MaxScanSize SIZE
+        Sets the maximum amount of data to be scanned for each input file.
+        Archives and other containers are recursively extracted  and  scanned
+        up to this value. The size of an archive plus the sum of the sizes of
+        all files within archive count toward the scan size. For example, a
+        1M uncompressed archive containing a single 1M inner file counts as
+        2M toward the max scan size.
 
-	StreamMaxLength** SIZE
-		Close the STREAM session when the data size limit is exceeded.
-		**Clamdscan Only
+    StreamMaxLength** SIZE
+        Close the STREAM session when the data size limit is exceeded.
+        **Clamdscan Only
 
-	Notes:
-		All options expressing a size are limited to max 2GB. Values in
-		excess will be reset to the maximum.
+    Notes:
+        All options expressing a size are limited to max 2GB. Values in
+        excess will be reset to the maximum.
 
 Configuration of ClamAV values
 ------------------------------
 
 Defaults in Archivematica are set to::
 
-	MaxFileSize=2000M
-	MaxScanSize=2000M
-	StreamMaxLength=2000M
+    MaxFileSize=2000M
+    MaxScanSize=2000M
+    StreamMaxLength=2000M
 
-	Where M is the unit Megabytes
+    Where M is the unit Megabytes
```

Comment:
Should we take this line out of the literal block to make it normal text?

Reply:
Hmmm good question! This is how it looks now: https://archivematica.org/en/docs/archivematica-1.15/admin-manual/installation-setup/customization/antivirus-admin/#configuration-of-clamav-values

Reply:
I decided that it would make sense to make it normal text :)